app/controllers/maintaining/contributor_teams/memberships_controller.rb

```diff
@@ -1,2 +1,59 @@
 class Maintaining::ContributorTeams::MembershipsController < ApplicationController
+  before_action :use_team
+  before_action :use_membership, only: %i[show edit update destroy]
+  before_action :use_seniority_options, only: %i[new create edit update]
+
+  def show; end
+
+  def new
+    @membership = ContributorTeam::Membership.new
+  end
+
+  def create
+    @membership = ContributorTeam::Membership.new(membership_params)
```

Comment:
If you do this:

```suggestion
    @membership = @team.memberships.build(membership_params)
```

Then you can remove the setting of the team in the santized params at the bottom of this file.